src/client/src/rt-components/loadable/Loadable.tsx

```diff
@@ -44,9 +44,9 @@ const Loadable: React.FC<Props> = ({
   minHeight,
 }) => {
   useEffect(() => {
-    onMount && onMount()
+    if (status === ServiceConnectionStatus.CONNECTED) onMount && onMount()
```

Comment:
I am not sure I quite parse this statement?

The `onMount` is already a single-line if statement, as in `if (onMount !== undefined) onMount()`

Should it be `if ((status === ServiceConnectionStatus.CONNECTED) && onMount) onMount()`?

Reply:
yes, I think you're right